packages/pages/src/dev/server/server.ts

```diff
@@ -40,9 +45,18 @@ export const createServer = async (
 
   // initialize the default project structure and use to help configure the dev server
   const projectStructure = await ProjectStructure.init({ scope });
+  const postCssPath = await getPostCssPathFromModuleName(
```

Comment:
All of the widget stuff should be contained. We shouldn't even load the index page. We should actually just pop open a tab at the widget path.

```
if (widget) {
 // do the widget things

 // we need some of the app.use stuff, but only for vite, none of our stuff
  app.use(vite.middlewares);
  app.use(errorMiddleware(vite));
  app.listen(devServerPort, () =>
    process.stdout.write(`listening on :${devServerPort}\n`)
  );
 return; // idk if return is required or app.listen() just stops further execution
}

// everything else we have
```